todoman/ui.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, todo, databases, formatter):
         self._help_text = urwid.Text(
             '\n\nGlobal:\n'
             ' F1: Toggle help\n'
-            ' Ctrl-C: Cancel\n\n'
+            ' Ctrl-C: Cancel\n'
+            ' Ctrl-S: Save (only works if not a shell shortcut already)\n\n'
```

Comment:
The thing is, it will be broken by default for almost everyone (`zsh` also binds that by default). I kinda dislike the idea of having a hotkey that won't work by default for almost everyone (we can make them configurable if you *really* want to use <kbd>ctrl</kbd> + <kbd>s</kbd>).